src/core/random.js

```diff
@@ -6,7 +6,87 @@
  * https://opensource.org/licenses/MIT.
  */
 
-import seedrandom from 'seedrandom';
+// the following code is the inlined alea PRNG from the seedrandom library
+function Alea(seed) {
+  var me = this,
+    mash = Mash();
+
+  me.next = function() {
+    var t = 2091639 * me.s0 + me.c * 2.3283064365386963e-10; // 2^-32
+    me.s0 = me.s1;
+    me.s1 = me.s2;
+    return (me.s2 = t - (me.c = t | 0));
+  };
+
+  // Apply the seeding algorithm from Baagoe.
+  me.c = 1;
+  me.s0 = mash(' ');
+  me.s1 = mash(' ');
+  me.s2 = mash(' ');
+  me.s0 -= mash(seed);
+  if (me.s0 < 0) {
+    me.s0 += 1;
+  }
+  me.s1 -= mash(seed);
+  if (me.s1 < 0) {
+    me.s1 += 1;
+  }
+  me.s2 -= mash(seed);
+  if (me.s2 < 0) {
+    me.s2 += 1;
+  }
+  mash = null;
+}
+
+function copy(f, t) {
+  t.c = f.c;
+  t.s0 = f.s0;
+  t.s1 = f.s1;
+  t.s2 = f.s2;
+  return t;
+}
+
+function Mash() {
+  var n = 0xefc8249d;
+
+  var mash = function(data) {
+    data = data.toString();
+    for (var i = 0; i < data.length; i++) {
+      n += data.charCodeAt(i);
+      var h = 0.02519603282416938 * n;
+      n = h >>> 0;
+      h -= n;
+      h *= n;
+      n = h >>> 0;
+      h -= n;
+      n += h * 0x100000000; // 2^32
+    }
+    return (n >>> 0) * 2.3283064365386963e-10; // 2^-32
+  };
+
+  return mash;
+}
+
+// this is the function that uses Alea to create a variety of methods.
+// the commented-out code was not used and lowered code coverage.
+function impl(seed, opts) {
+  var xg = new Alea(seed),
+    state = opts && opts.state,
+    prng = xg.next;
+  // prng.int32 = function() { return (xg.next() * 0x100000000) | 0; }
```

Comment:
If we don't envision ourselves using these in the future, let's just remove them rather than leaving them commented out. Alternatively, you could use `prng.int32` in the die roll code below with the modulus operator.

Reply:
I've dropped them - I think one primitive to generate random numbers inside the library is sufficient.